python-package/lightgbm/sklearn.py

```diff
@@ -719,10 +720,9 @@ def predict(self, X, raw_score=False, start_iteration=0, num_iteration=None,
             X = _LGBMCheckArray(X, accept_sparse=True, force_all_finite=False)
         n_features = X.shape[1]
         if self._n_features != n_features:
-            raise ValueError("Number of features of the model must "
-                             "match the input. Model n_features_ is %s and "
-                             "input n_features is %s "
-                             % (self._n_features, n_features))
+            raise ValueError(f"Number of features of the model must "
```

Comment:
```suggestion
            raise ValueError("Number of features of the model must "
```

This first string is a string literal without any templating, so it doesn't require an `f`